docs/tutorials/quickstart-mdldev.rst

```diff
@@ -183,8 +182,8 @@ schedules to run.
 
 #. Enter the cluster address in the browser address bar to view experiment progress in the WebUI. If
    you installed locally using the ``det deploy local`` command, the URL is
-   ``http://localhost:8080/``. Accept the default ``determined`` username and click **Sign In**. No
-   password is required.
+   ``http://localhost:8080/``. Accept ``determined`` as the default username and click **Sign In**.
+   A password is not required.
```

Comment:
we have a shared snippet for these bits now... can we just use the shared snippet in more places?